src/Plugin/HistoryPlugin.php

```diff
@@ -40,8 +40,10 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
             $journal->addSuccess($request, $response);
 
             return $response;
-        }, function (Exception $exception) use ($request, $journal) {
-            $journal->addFailure($request, $exception);
+        }, function (\Exception $exception) use ($request, $journal) {
+            if ($exception instanceof Exception) {
+                $journal->addFailure($request, $exception);
```

Comment:
I guess we can't. Or we shouldn't. The journal is only meant to HTTP related exceptions.